### PR TITLE
Felinids can no longer contract organ failure by licking wounds

### PR DESCRIPTION
**Pull request by Ryll-Ryll**
## About The Pull Request
This one's a simple goof on my part. Felinids licking wounds would forcibly contract every disease the patient had, which included things like appendicitis and myocardial infarction. This fixes that by adding the same checks that being exposed to someone's blood directly runs, filtering out diseases marked as DISEASE_SPREAD_SPECIAL or DISEASE_SPREAD_NON_CONTAGIOUS

## Why It's Good For The Game
Prevents an entire generation of youths from drastically misunderstanding how diseases work in real life

## Changelog
:cl: Ryll/Shaps
fix: Felinids can no longer contract heart attacks and appendicitis from licking the wounds of the injured.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
oh god you can dupe ghost poly with this trick

**Comment by msgerbs** [comment]
> 
> 
> oh god you can dupe ghost poly with this trick
> 
> wait, looking at the code again, I don't think you can, darn.

Wait... What??

**Comment by Winterous** [comment]
Does this mean felinids can contract snail disease?

**Comment by necromanceranne** [comment]
They probably contact all the transformation diseases.

**Comment by AtlasRules** [comment]
they deserve it for playing felinid, anyway

File: code/datums/wounds/slash.dm [hunk 1]
```diff
@@ -161,8 +161,10 @@
 /// if a felinid is licking this cut to reduce bleeding
 /datum/wound/slash/proc/lick_wounds(mob/living/carbon/human/user)
 	// transmission is one way patient -> felinid since google said cat saliva is antiseptic or whatever, and also because felinids are already risking getting beaten for this even without people suspecting they're spreading a deathvirus
-	for(var/datum/disease/D in victim.diseases)
-		user.ForceContractDisease(D)
+	for(var/datum/disease/iter_disease in victim.diseases)
+		if((iter_disease.spread_flags & DISEASE_SPREAD_SPECIAL) || (iter_disease.spread_flags & DISEASE_SPREAD_NON_CONTAGIOUS))
```
Comment (Rohesie) [inline code/datums/wounds/slash.dm:165]:
```suggestion
	for(var/d in victim.diseases)
		var/datum/disease/iter_disease = d
		if(iter_disease.spread_flags & (DISEASE_SPREAD_SPECIAL | DISEASE_SPREAD_NON_CONTAGIOUS))
```

**Comment by brotherbeyondd** [comment]
You fixed it! How dare you!

**Pull request merged**